src/BeautifulComments/ClyRichTextClassCommentEditorToolMorph.class.st

```diff
@@ -26,12 +26,9 @@ ClyRichTextClassCommentEditorToolMorph class >> classCommentTabActivation [
 { #category : 'operations' }
 ClyRichTextClassCommentEditorToolMorph >> applyChanges [
 
-	self pendingText asString = self editingText asString 
-		ifFalse: [ editingClass 
-						comment: self pendingText 
-						stamp: Author changeStamp].
+	self pendingText asString = self editingText asString ifFalse: [ editingClass comment: self pendingText stamp: (Date today mmddyyyy, ' ' , ((String streamContents: [:s | Time now print24: true on: s]) copyFrom: 1 to: 5)) ].
```

Comment:
should reuse the `#editionTimestamp`method that needs to go  in `ClyTextEditorToolMorph` class